dali/python/backend_impl.cc

```diff
@@ -1471,27 +1532,23 @@ void ExposeTensorList(py::module &m) {
         layout : str
               Layout of the data
         )code")
-    .def(py::init([](TensorList<GPUBackend> *tl, py::object layout) {
+    .def(py::init([](TensorList<GPUBackend> *tl, std::optional<std::string> layout) {
```

Comment:
```suggestion
    .def(py::init([](TensorList<GPUBackend> *tl, std::optional<std::string> layout = {}) {
```
To be consistent with above?